code/parse/sexp/LuaSEXP.cpp

```diff
@@ -38,7 +39,14 @@ SCP_unordered_map<SCP_string, int> parameter_type_mapping{{ "boolean",      OPF_
 														  { "wing",         OPF_WING },
 														  { "shipclass",    OPF_SHIP_CLASS_NAME },
 														  { "weaponclass",  OPF_WEAPON_NAME },
-														  { "soundentry",   OPF_GAME_SND }, };
+														  { "soundentry",   OPF_GAME_SND }, 
+														  { "oswpt-ship+waypoint",   OPF_SHIP_POINT },
+														  { "oswpt-ship+wing",   OPF_SHIP_WING },
+														  { "oswpt-ship+wing+team",   OPF_SHIP_WING_WHOLETEAM },
+														  { "oswpt-ship+wing+team+waypoint",   OPF_SHIP_WING_SHIPONTEAM_POINT },
+														  { "oswpt-ship+wing+waypoint",   OPF_SHIP_WING_POINT },
+														  { "oswpt-ship+wing+waypoint+none",   OPF_SHIP_WING_POINT_OR_NONE }, };
```

Comment:
You could remove the `oswpt-` prefix and just declare the scripting type to be the data it accepts